### PR TITLE
DISTMYSQL-358: Tags are not displayed for unhealthy instances

### PR DESCRIPTION
**Pull request by kamil-holubicki**
https://jira.percona.com/browse/DISTMYSQL-358

Problem:
DISTMYSQL-169 added display of instance tags on GUI, however, tags are not displayed for unhealthy instances.

Cause:
Unhealthy instances display is driven from another callstack (display of unhealthy instances didn't consider tags)

Solution:
Move tags display logic into the common code.

### CODE REVIEW

**Comment by fabio-silva** [comment]
@kamil-holubicki As you may notice, I have no relation to this repo, so there is no reason to tag me here. Thanks

File: resources/public/js/cluster.js [hunk 1]
```diff
@@ -1546,25 +1546,27 @@ function Cluster() {
...
-        }
-        if (replica.notRecentlyCheckedProblem()) {
-          return
-        }
-        recoveryListing.append(
-          '<li><a href="#" data-btn="recover-suggested-successor" data-command="recover-suggested-successor" data-successor-host="' + replica.Key.Hostname + '" data-successor-port="' + replica.Key.Port + '">Recover, try to promote <code>' + replica.title + '</code></a></li>');
-      });
+      if (instance.children) {
+        instance.children.forEach(function(replica) {
+          if (!replica.LogBinEnabled) {
```
Comment (fabio-silva) [inline resources/public/js/cluster.js:1551]: these conditions can be merged into multiple OR statements

File: resources/public/js/orchestrator.js [hunk 2]
```diff
@@ -973,6 +973,18 @@ function renderInstanceElement(popoverElement, instance, renderType) {
       popoverElement.find("h3 div.pull-right").prepend('<span class="glyphicon glyphicon-volume-off" title="' + downtimeMessage + '"></span> ');
     }
 
+    $.get(appUrl("/api/tags/" + instance.Key.Hostname + "/" + instance.Key.Port), function(tagStrings) {
+      if (tagStrings.length) {
+        // Need to remember this. It is used in instance's modal window.
+        instance.tagStrings = tagStrings;
+        var tagsText = "";
+        instance.tagStrings.forEach(function(tag){
```
Comment (fabio-silva) [inline resources/public/js/orchestrator.js:981]:
```suggestion
        instance.tagStrings.forEach(function(tag) {
```

**Review by fabio-silva — CHANGES_REQUESTED** [review]

File: resources/public/js/cluster.js (hunk 1 above)
Comment (kamil-holubicki) [reply resources/public/js/cluster.js:1551]: Agree, but this was the original, upstream code, so it is desired to keep it with minimal changes.

File: resources/public/js/orchestrator.js (hunk 2 above)
Comment (kamil-holubicki) [reply resources/public/js/orchestrator.js]: corrected

**Pull request merged**